samples/automl/automlTranslationDataset.js

```diff
@@ -150,41 +149,39 @@ function getDataset(projectId, computeRegion, datasetId) {
   // const datasetId = `Id of the dataset`;
 
   // Get the full path of the dataset.
-  const datasetFullId = client.datasetPath(projectId, computeRegion, datasetId);
+  const datasetFullId = await client.datasetPath(

```

Comment:
I don't think this was returning a promise. Can probably just drop the `await` :)